app/models/transaction_type.rb

```diff
@@ -26,6 +26,10 @@ def account_initializer
       first
     end
 
+    def initialize
+      find_by(name: INITIALIZE)
```

Comment:
Change to 
```
@@initialize = find_by(name: INITIALIZE)
```

Do this to the other class methods as well.
* `def account_initializer`
* `def income`
* `def salary`

Reply:
Type of variables in a class source code

1. `@@i_am_a_class_variable`
2. `@i_am_an_instance_variable`
3. `i_am_a_local_variable`